lib/backup_restore/backuper.rb

```diff
@@ -192,81 +191,58 @@ def pg_dump_command
         "--no-owner",                 # do not output commands to set ownership of objects
         "--no-privileges",            # prevent dumping of access privileges
         "--verbose",                  # specifies verbose mode
+        "--compress=4",               # Compression level of 4
         host_argument,                # the hostname to connect to (if any)
         port_argument,                # the port to connect to (if any)
         username_argument,            # the username to connect as (if any)
         db_conf.database              # the name of the database to dump
       ].join(" ")
     end
 
-    def update_dump
-      log "Updating dump for more awesomeness..."
-
-      `#{sed_command}`
-    end
-
-    def sed_command
```

Comment:
Let's not mess with the valid copy of the `pg_dump` file. We'll do the `sed`ing when restoring.